src/hooks/useAuthForm.tsx

```diff
@@ -18,17 +24,71 @@ export interface CommonFieldValues extends FieldValues {
   passwordConfirmation: string;
 }
 const useAuthForm = () => {
+  const dispatch = useDispatch();
```

Comment:
need to add ts to the RTK, please read the doc

Reply:
TS generally added to RTK